.vscode/settings.json

```diff
@@ -2,5 +2,8 @@
   "editor.semanticHighlighting.enabled": true,
   "[typescript]": {
     "editor.defaultFormatter": "denoland.vscode-deno"
+  },
+  "[json]": {
+    "editor.defaultFormatter": "denoland.vscode-deno"
   }
```

Comment:
Wasn't really necessary in this PR, but I just want to make sure prettier doesn't conflict with the formatting I'm expecting the file to have.